openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -164,6 +164,11 @@ func (e *extension) Reconcile(ctx context.Context, comp base.KComponent) error {
 		common.ConfigureIfUnset(&ks.Spec.CommonSpec, monitoring.ObservabilityCMName, monitoring.ObservabilityBackendKey, "none")
 	}
 
+	// DomainMapping cleanup
+	if err := e.cleanupDomainMapping(ctx, ks); err != nil {
+		//TODO: we don't need to propagate the error further?
+		println("Failed to cleanup domain mapping:", err)
```

Comment:
Is there a logger to use?

Reply:
If I remember correctly	`logger := logging.FromContext(ctx)`